### ISSUE TITLE
just an idea

### ISSUE THREAD

**Issue by PatrickHonorez** [GustavBrock/VBA.Timezone-Windows#1]
Hi Gustav,

I had the same problem, to solve and in the context where I was, I found a very effective and simple solution, using OUtlook built-in support for time zones..  
See https://stackoverflow.com/a/46052360/78522

Regards,

Patrick

**Comment by GustavBrock** [comment]
Yes, thank you @PatrickHonorez, I've seen that (note my comment dated 2020-06-20, please).
However, as noted, it requires _Outlook_ to be installed, which not always is the case.
